stdlib/SparseArrays/src/higherorderfns.jl

```diff
@@ -1154,4 +1155,23 @@ map(f::Tf, A::SparseOrStructuredMatrix, Bs::Vararg{SparseOrStructuredMatrix,N})
 map!(f::Tf, C::AbstractSparseMatrixCSC, A::SparseOrStructuredMatrix, Bs::Vararg{SparseOrStructuredMatrix,N}) where {Tf,N} =
     (_checksameshape(C, A, Bs...); _noshapecheck_map!(f, C, _sparsifystructured(A), map(_sparsifystructured, Bs)...))
 
+
+# (13) extrema methods optimized for sparse vectors/matrices.
+function extrema(f, A::SparseVecOrMat)
+    if iszero(length(A))
```

Comment:
In the spirit of

https://github.com/JuliaLang/julia/blob/dbd3d4cd07e54cc665eae80485fb07394266ad57/stdlib/SparseArrays/src/sparsevector.jl#L1360-L1367

you may want to introduce a `M = length(A)` and reuse it below. As I said, I wonder whether we should move those here and extend them to `SparseVecOrMat`. Hopefully, this doesn't introduce method ambiguities, but can be done in a separate PR.

Reply:
Done.
Concerning `minimum` and `maximum`, they definitely need to be moved and applied on `SparseVecOrMat`. There are no optimizations for sparse matrices on either of these functions for now.